### PR TITLE
In 148611067 only a sensitive person with permission

### PR DESCRIPTION
**Pull request by jgedeon-osi**

### CODE REVIEW

File: app/controllers/concerns/authentication_concern.rb [hunk 1]
```diff
@@ -11,10 +11,22 @@ module AuthenticationConcern # :nodoc:
     private
 
     def authenticate_request
-      token = request.headers['Authorization']
-      response = Faraday.get("#{ENV.fetch('AUTHENTICATION_URL')}/authn/validate?token=#{token}")
-      return if response.status == 200
-      render json: { errors: ['Forbidden, request not authorized'] }, status: 403
+      @security_token = request.headers['Authorization']
+      return if @security_token.nil?
+
```
Comment (NateOSI) [inline app/controllers/concerns/authentication_concern.rb]: excess new lines

File: spec/models/ability_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        'roles' => ['Supervisor'],
+        'county_code' => 23,
+        'county_name' => 'Manteca',
+        'privileges' => [
+          'Countywide Read/Write'
+        ]
+      }
+    end
+
+    it 'allows access' do
```
Comment (ThomasOSI) [inline spec/models/ability_spec.rb]: disallows access?

File: docker-compose.yml [hunk 3]
```diff
@@ -12,8 +12,11 @@ services:
       - db
     ports:
       - "3001:3000"
+    volumes:
```
Comment (ThomasOSI) [inline docker-compose.yml]: Why do we want a volume for API?

Comment (jgedeon-osi) [reply docker-compose.yml]: I was coding in vim outside the container but running tests inside the container, so I needed the volume to have access to my edits in the container for tests.

File: app/controllers/concerns/authentication_concern.rb (hunk 1 above)
Comment (jgedeon-osi) [reply app/controllers/concerns/authentication_concern.rb]: Removed the excess newlines.

File: spec/models/ability_spec.rb (hunk 2 above)
Comment (jgedeon-osi) [reply spec/models/ability_spec.rb]: changed.

**Comment by jgedeon-osi** [comment]
Added missing request spec and addressed comments.

**Pull request merged**